tox.ini

```diff
@@ -4,8 +4,9 @@ project = yelp_bytes
 envlist = py27,py34,py35,pypy,pypy3
 
 [testenv]
-install_command = pip install --use-wheel {opts} {packages}
 deps = -rrequirements-dev.txt
+setenv =
+    PIP_ONLY_BINARY=:all:
```

Comment:
the `--use-wheel` was an obsolete pip 1.4 setting which enabled (at the time) experimental wheel usage -- these lines shouldn't be necessary